storage/s3.py

```diff
@@ -14,9 +14,12 @@
 def available(cache={}):
     try:
         if not cache:
+            print(DEST_BUCKET)
+            print(fs.ls(DEST_BUCKET))
             cache.update({os.path.basename(p).split(".")[0]: p for p in fs.ls(DEST_BUCKET)})
     except FileNotFoundError:
         cache = {}
+    print(cache)
```

Comment:
... and neither this print should be here.

Reply:
oooh. those darn debugging prints...